minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -511,6 +590,32 @@ public Builder addProperties(Map<PropertyKey, String> properties) {
       return this;
     }
 
+    /**
+     * Sets master specific properties.
+     * The keys of the properties are the indexes of masters
+     * which are numbers between 0 to the number of masters(not included).
```

Comment:
there should be spaces before parentheses in english
instead of "not inclusive", you can just say "exclusive"

Reply:
Changed, thanks!